dsgrid/config/project_config.py

```diff
@@ -461,7 +470,9 @@ def add_dataset_dimension_mappings(self, dataset_config, references):
     def check_dataset_dimension_mappings(
```

Comment:
Is this still needed?

Reply:
I say delete it. The other checks that look at all records will catch anything caught here.